### DISCUSSION TITLE
[Restrictions] Revoir l'affichage des lycées sélectionnés comme pouvant participer à une immersion

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#13] [4- Saisie de l'offre d'immersion]
Certains utilisateurs rencontrent des difficultés d'affichage lorsqu'ils essaient de sélectionner des lycées dans les restrictions pour un créneau d'immersion.

**Comment by efinkbeiner5** [comment]
Liste déroulante trop courte, revoir l'affichage, sera fait car relève de l'exploitation
Chiffrage : faible

**Comment by efinkbeiner5** [comment]
Bug corrigé en V3
